orderbook/src/database/orders.rs

```diff
@@ -201,8 +201,51 @@ struct OrdersQueryRow {
     signing_scheme: DbSigningScheme,
 }
 
+/// Amount which is the minimum to consider an order fully filled.
+/// original amount / FILLED_PERCENTAGE_DENOMINATOR => minimum amount
+/// Fully filled order => amount - executed_amount < minimum_amount
+/// The higher this number, the smaller the amount to consider an order filled.
+const FILLED_PERCENTAGE_DENOMINATOR: u32 = 10_000; // 10_000 == 0.01 %
+
+/// Based on given amounts and a constant fill percentage,
+/// determines whether given difference is enough for considering
+/// the order as fully filled
+fn is_fully_filled(amount: &BigDecimal, executed_amount: &BigDecimal) -> bool {
+    if executed_amount.is_zero() {
+        return false;
+    }
+    let negligible_amount = amount / FILLED_PERCENTAGE_DENOMINATOR;
+    let difference = amount - executed_amount;
+
+    difference < negligible_amount
+}
```

Comment:
Also not very happy with this being inside the database folder.
Feels more like an Order model than SQL related. Any suggestions?

Reply:
I see what you mean but I also think it's practical to keep this here for now. I feel the function makes the most sense somewhere in the orderbook crate though.

Reply:
Removed this logic altogether